### PR TITLE
Assets/avatars

### PR DESCRIPTION
**Pull request by Kamieniu**
## Description
Added assets as a local files. 
All Seeds are targeting local files.

Also added .env file to manage ports and host, so candidate can setup it before he launch it and define which port he wants to use.

Replace file paths from Google Drive to local ones.

It's a pure MAGIC :xd

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,2 @@
+PORT=4000
+HOST=http://localhost:
```
Comment (kulek1) [inline .env]: What do you think about having just `http://localhost` without `:` for port?
You could also create a util for getting these two properties e.g. `getServerURL()` or something like that and use it in `seed/avatars.ts`.

Comment (Kamieniu) [reply .env]: There should be a port as React/Angular or Vue also runs on localhost.

File: src/server-graphql.ts [hunk 2]
```diff
@@ -1,10 +1,16 @@
...
+import bodyParser from 'body-parser-graphql';
+import dotenv from 'dotenv';
+import express, { Request } from 'express';
 import { GraphQLServer } from 'graphql-yoga';
 import { prisma } from '../generated/prisma-client';
 import { resolvers } from './graphql/resolvers';
 
+dotenv.config();
+
+const { HOST } = process.env;
```
[the remark this answers is not in the document]
Comment (qmixi) [reply src/server-graphql.ts]: Screenshot 2019-11-17 at 22 00 29

File: tsconfig.json [hunk 3]
```diff
@@ -2,6 +2,12 @@
     "compilerOptions": {
         "lib": ["es2016", "esnext.asynciterable"],
         "baseUrl": "./",
+        "module": "commonjs",
+        "esModuleInterop": true,
+        "target": "es6",
+        "noImplicitAny": true,
```
Comment (qmixi) [inline tsconfig.json]: @Kamieniu I recommend adding this option with types support in a separate PR. With current changes application is not building due to Types errors for both `REST` and `GraphQL` APIs.
Screenshot 2019-11-17 at 21 59 56

**Pull request merged**